style.css

```diff
@@ -40,4 +40,125 @@ button {
 
 .textError {
     color: red;
+}
+
+.repo-card svg {
+    width: 15px;
+}
+
+a {
+    text-decoration: none;
+}
+
+/* Repo-card classes */
+
+.repo-card {
+    border: 1px solid var(--card-border);
+    border-radius: 0.3em;
+    background-color: transparent;
+    color: var(--card-text);
+    padding: 1em;
+}
+
+.repo-link {
+    color: var(--card-text);
```

Comment:
Here you can do `color: inherit`. This way, it will have the same color as the parent, in this case `repo-card`, so you don't need to duplicate it.